probe/endpoint/conntrack.go

```diff
@@ -15,8 +15,9 @@ import (
 )
 
 const (
-	modules          = "/proc/modules"
-	conntrackModule  = "nf_conntrack"
+	// From https://www.kernel.org/doc/Documentation/networking/nf_conntrack-sysctl.txt
+	// Check a tcp-related file for existence since we need tcp tracking
+	procFileToCheck  = "sys/net/netfilter/nf_conntrack_tcp_timeout_close"
```

Comment:
It's been available at least since 2.6.20 https://github.com/torvalds/linux/blame/a7fd20d1c476af4563e66865213474a2f9f473a4/net/netfilter/nf_conntrack_proto_tcp.c#L1441